broker/tests/unit/plugin/kubejobs/test_plugin.py

```diff
@@ -174,11 +191,83 @@ def test_update_monitor_info(self):
                                       queue_size, redis_ip, redis_port)
         self.assertEqual(data.get('monitor_info'), monitor_info_after)
 
-    """
-    Test the Get and Update Application State of
-    the Kubejobs plugin.
-    """
+    def test_start_visualization(self):
```

Comment:
Please set kj-000001 to a constant and reuse that for better readability

Reply:
Done